Sources/Purchasing/StoreKit1/StoreKit1Wrapper.swift

```diff
@@ -112,6 +114,34 @@ class StoreKit1Wrapper: NSObject {
         return payment
     }
 
+    private func notifyDelegateOfExistingTransactions() {
```

Comment:
Should we add something to the name of this method to indicate that it's only going to be run on tests? Like `notifyDelegateOfExistingTransactionsInTests`?

Reply:
Good idea. Maybe `notifyDelegateOfExistingTransactionsIfNeeded`